mkdocs.yml

```diff
@@ -0,0 +1,72 @@
+theme:
+  name: material
+  features:
+    - navigation.sections
+  palette:
+    - scheme: default
+      primary: white
+      accent: amber
+      toggle:
+        icon: material/weather-night
+        name: Switch to dark mode
+    - scheme: slate
+      primary: black
+      accent: amber
+      toggle:
+        icon: material/weather-sunny
+        name: Switch to light mode
+  font:
+    text: Lato
+  icon:
+    repo: fontawesome/brands/github
+
+site_name: SciCookie
+site_description: Documentation for SciCookie
+site_author: OpenScienceLabs Team
+site_url: https://osl-incubator.github.io/scicookie
+
+repo_url: https://github.com/osl-incubator/scicookie
+repo_name: osl-incubator/scicookie
+edit_uri: ./edit/main/docs
+
+extra_css:
+  - stylesheets/extra.css
```

Comment:
I don' t think we have this css file? Should this line be removed?

Reply:
yes, I must've missed it while experimenting